### PR TITLE
Login Validation

### PR DESCRIPTION
**Pull request by Alekhyagnv**
Checking if the user already exists in the database. If the users exist, we are comparing the given password and the password in the database and then generating the token of the user.

### CODE REVIEW

File: bd/tryLogin.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package bd
+
+import (
+	"github.com/SaiKumarMalve/Gator-News/models"
+	"golang.org/x/crypto/bcrypt"
+)
+
```
Comment (SaiKumarMalve) [inline bd/tryLogin.go:7]: Write a brief comment of what function does for maintainability purpose

**Review by SaiKumarMalve — CHANGES_REQUESTED** [review]
I don't see a token generator here. Can you push that code?

File: bd/tryLogin.go (hunk 1 above)
Comment (Alekhyagnv) [reply bd/tryLogin.go:7]: I will write the comments.

**Review by SaiKumarMalve — APPROVED** [review]
Looks good to me. Merge the changes

**Pull request merged**